docs/src/main/asciidoc/logging.adoc

```diff
@@ -183,6 +283,8 @@ quarkus.log.category."io.quarkus.smallrye.jwt".level=TRACE
 quarkus.log.category."io.undertow.request.security".level=TRACE
 ----
 
+NOTE: as we don't change the root logger, console log will only contains `INFO` or higher order logs.
```

Comment:
I think most admonitions start with capital. Regardless of style I think it should be consistent.

Reply:
Done